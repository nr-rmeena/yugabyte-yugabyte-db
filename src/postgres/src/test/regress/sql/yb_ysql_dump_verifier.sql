-- tables
 \d
-- tablespaces
 \db
-- users
 \du
-- namespaces
 \dn
-- tablegroups
 \dgr+
-- tablegroups & tables
 \dgrt+
