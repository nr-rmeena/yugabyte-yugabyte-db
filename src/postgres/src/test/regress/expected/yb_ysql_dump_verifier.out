                               List of relations
 Schema |                 Name                  |   Type   |       Owner
--------+---------------------------------------+----------+-------------------
 public | chat_user                             | table    | yugabyte_test
 public | rls_private                           | table    | yugabyte_test
 public | rls_public                            | table    | yugabyte_test
 public | table1                                | table    | yugabyte_test
 public | table2                                | table    | yugabyte_test
 public | tbl1                                  | table    | yugabyte_test
 public | tbl10                                 | table    | yugabyte_test
 public | tbl11                                 | table    | yugabyte_test
 public | tbl12                                 | table    | yugabyte_test
 public | tbl13                                 | table    | yugabyte_test
 public | tbl1_a_seq                            | sequence | yugabyte_test
 public | tbl2                                  | table    | yugabyte_test
 public | tbl2_a_seq                            | sequence | yugabyte_test
 public | tbl3                                  | table    | yugabyte_test
 public | tbl4                                  | table    | yugabyte_test
 public | tbl5                                  | table    | yugabyte_test
 public | tbl6                                  | table    | yugabyte_test
 public | tbl7                                  | table    | yugabyte_test
 public | tbl8                                  | table    | yugabyte_test
 public | tbl9                                  | table    | yugabyte_test
 public | this_table_is_just_to_shift_oids_by_1 | table    | yugabyte_test
 public | this_table_is_just_to_shift_oids_by_2 | table    | yugabyte_test
 public | this_table_is_just_to_shift_oids_by_3 | table    | yugabyte_test
 public | uaccount                              | table    | regress_rls_alice
(24 rows)

          List of tablespaces
    Name    |     Owner     | Location
------------+---------------+----------
 pg_default | postgres      |
 pg_global  | postgres      |
 x          | yugabyte_test |
 z          | yugabyte_test |
(4 rows)

                                       List of roles
     Role name     |                         Attributes                         | Member of
-------------------+------------------------------------------------------------+-----------
 postgres          | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
 regress_rls_alice | Cannot login                                               | {}
 rls_user          | Cannot login                                               | {}
 yb_extension      | Cannot login                                               | {}
 yb_fdw            | Cannot login                                               | {}
 yugabyte          | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
 yugabyte_test     | Superuser, Create role, Create DB, Bypass RLS              | {}

  List of schemas
  Name  |  Owner
--------+----------
 public | postgres
(1 row)

                          List of tablegroups
 Name | Owner | Access privileges | Description | Tablespace | Options
------+-------+-------------------+-------------+------------+---------
(0 rows)

                                                             List of tablegroup tables
 Group Name | Group Owner | Access privileges | Group Description | Group Tablespace | Group Options | Name | Type | Owner | Rel Description | Size
------------+-------------+-------------------+-------------------+------------------+---------------+------+------+-------+-----------------+------
(0 rows)
